src/ast/nodes/ConditionalExpression.ts

```diff
@@ -101,7 +101,9 @@ export default class ConditionalExpression extends NodeBase {
 
 	include() {
 		this.included = true;
-		const testValue = this.hasUnknownTestValue ? UNKNOWN_VALUE : this.getTestValue();
+		const testValue = this.hasUnknownTestValue
+			? UNKNOWN_VALUE
+			: this.getTestValue(NEW_EXECUTION_PATH);
```

Comment:
Is there definitely no risk that these NEW_EXECUTION_PATH constants will inadvertently get paths assigned or checked resulting in unwanted mutations for other paths using it?

Reply:
Not for now as everything is immutable i.e. when a value is added, everything is copied (this is what we have immutable.js for). It's probably NOT a very efficient solution. I hope to completely refactor it soon to make it more efficient and get rid of immutable.js.